camelot/admin/object_admin.py

```diff
@@ -358,6 +359,12 @@ def get_form_actions( self, obj=None ):
             states to make the appearance of actions dynamic on a form)
         :return: a list of :class:`camelot.admin.action.base.Action` objects
         """
+        # Check __facade_args__ for 'show_actions'
+        if isinstance(self.entity, EntityMeta):
```

Comment:
dit moet hier nt opgelost worden, het is niet de bedoeling dat je het displayen van actions kan overrulen.

ofwel moeten er geen actions in de admin zitten, ofwel moet de admin.get_state conditioneel zijn